sources/packages/backend/workflow/BPMN/camunda-8/fulltime-assessment-2021-2022.bpmn

```diff
@@ -1743,54 +1621,233 @@ or (calculatedDataPartnerFedralProvincialPDReceiptCost &gt; 0)</bpmn:conditionEx
+    <bpmn:intermediateThrowEvent id="Event_105ptqe" name="Calculate Family Income">
+      <bpmn:extensionElements>
+        <zeebe:ioMapping>
+          <zeebe:output source="=calculatedDataStudentTotalIncome" target="calculatedDataTotalFamilyIncome" />
+        </zeebe:ioMapping>
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_00b4sbw</bpmn:incoming>
+      <bpmn:outgoing>Flow_0ug361k</bpmn:outgoing>
+    </bpmn:intermediateThrowEvent>
+    <bpmn:intermediateThrowEvent id="Event_0zn13zd" name="Calculate Family Income">
+      <bpmn:extensionElements>
+        <zeebe:ioMapping>
+          <zeebe:output source="= calculatedDataStudentTotalIncome + calculatedDataPartner1TotalIncome" target="calculatedDataTotalFamilyIncome" />
+        </zeebe:ioMapping>
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_189jv8b</bpmn:incoming>
+      <bpmn:outgoing>Flow_1ydy4cj</bpmn:outgoing>
+    </bpmn:intermediateThrowEvent>
+    <bpmn:exclusiveGateway id="Gateway_1qh6zrs">
+      <bpmn:incoming>Flow_1qca6yi</bpmn:incoming>
+      <bpmn:incoming>Flow_1ydy4cj</bpmn:incoming>
+      <bpmn:incoming>Flow_0ug361k</bpmn:incoming>
+      <bpmn:outgoing>Flow_09cjv05</bpmn:outgoing>
+    </bpmn:exclusiveGateway>
+    <bpmn:intermediateThrowEvent id="Event_177w8t8" name="calculatedDataFamilySize">
+      <bpmn:extensionElements>
+        <zeebe:ioMapping>
+          <zeebe:output source="=calculatedDataTotalParentEligibleDependants + studentDataNumberOfParents + 1" target="calculatedDataFamilySize" />
+        </zeebe:ioMapping>
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_03s68ur</bpmn:incoming>
+      <bpmn:outgoing>Flow_05fykma</bpmn:outgoing>
+    </bpmn:intermediateThrowEvent>
+    <bpmn:intermediateThrowEvent id="Event_0bdny14" name="calculatedDataFamilySize">
+      <bpmn:extensionElements>
+        <zeebe:ioMapping>
+          <zeebe:output source="=calculatedDataTotalEligibleDependants + 1" target="calculatedDataFamilySize" />
+        </zeebe:ioMapping>
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_0otihoy</bpmn:incoming>
+      <bpmn:outgoing>Flow_00b4sbw</bpmn:outgoing>
+    </bpmn:intermediateThrowEvent>
+    <bpmn:intermediateThrowEvent id="Event_1k4n9fq" name="calculatedDataFamilySize">
+      <bpmn:extensionElements>
+        <zeebe:ioMapping>
+          <zeebe:output source="=calculatedDataTotalEligibleDependants + 2" target="calculatedDataFamilySize" />
+        </zeebe:ioMapping>
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_1l0g9nv</bpmn:incoming>
+      <bpmn:outgoing>Flow_189jv8b</bpmn:outgoing>
+    </bpmn:intermediateThrowEvent>
+    <bpmn:intermediateThrowEvent id="Event_14jfj5g" name="calculatedDataTotalMSOLAllowance">
+      <bpmn:extensionElements>
+        <zeebe:ioMapping>
+          <zeebe:output source="=sum([&#10;     calculatedDataStudentMSOLAllowance + calculatedDataDependantTotalMSOLAllowance&#10;   ][item != null]&#10;)" target="calculatedDataTotalMSOLAllowance" />
```

Comment:
The syntax below with `sum` and `[item != null]` is used to execute the sum of multiple items when some can potentially be null. In this case, there is no benefit to using this approach (unless I am missing something).
The syntax can be simplified as below.
```
calculatedDataDependantMSOLAllowance * calculatedDataTotalEligibleDependants
````